include/SFML/Window/Keyboard.hpp

```diff
@@ -153,6 +158,158 @@ class SFML_WINDOW_API Keyboard
+        ScanNum9,              ///< Keyboard 9 and ) key
+        ScanNum0,              ///< Keyboard 0 and ) key
+        ScanEnter,             ///< Keyboard Return (ENTER) key
+        ScanEscape,            ///< Keyboard Escape key
+        ScanBackspace,         ///< Keyboard Backspace key
+        // TODO above it's BackSpace, but is it correct? What do we use here?
+        ScanTab,               ///< Keyboard Tab key
+        ScanSpace,             ///< Keyboard Space key
+        ScanHyphen,            ///< Keyboard - and _ key
+        ScanEquals,            ///< Keyboard = and +
+        ScanLBracket,          ///< Keyboard [ and { key
+        ScanRBracket,          ///< Keyboard ] and } key
+        ScanBackslash,         ///< Keyboard \ and | key
+        // TODO capitalisation
+        ScanDash,              ///< Keyboard Non-US # and ~
+        ScanSemicolon,         ///< Keyboard ; and : key
+        // TODO capitalisation
+        ScanQuote,             ///< Keyboard ' and " key
+        ScanGraveAccent,       ///< Keyboard ` and ~ key
+        ScanComma,             ///< Keyboard , and < key
+        ScanPeriod,            ///< Keyboard . and > key
+        ScanForwardSlash,      ///< Keyboard / and ? key
+        ScanF1,                ///< Keyboard F1 key
+        ScanF2,                ///< Keyboard F2 key
+        ScanF3,                ///< Keyboard F3 key
+        ScanF4,                ///< Keyboard F4 key
+        ScanF5,                ///< Keyboard F5 key
+        ScanF6,                ///< Keyboard F6 key
+        ScanF7,                ///< Keyboard F7 key
+        ScanF8,                ///< Keyboard F8 key
+        ScanF9,                ///< Keyboard F9 key
+        ScanF10,               ///< Keyboard F10 key
+        ScanF11,               ///< Keyboard F11 key
+        ScanF12,               ///< Keyboard F12 key
+        ScanF13,               ///< Keyboard F13 key
+        ScanF14,               ///< Keyboard F14 key
+        ScanF15,               ///< Keyboard F15 key
+        ScanCapsLock,          ///< Keyboard Caps Lock key
+        ScanPrintScreen,       ///< Keyboard Print Screen key
+        ScanScrollLock,        ///< Keyboard Scroll Lock key
+        ScanPause,             ///< Keyboard Pause key
+        ScanInsert,            ///< Keyboard Insert key
+        ScanHome,              ///< Keyboard Home key
+        ScanPageUp,            ///< Keyboard Page Up key
+        ScanDelete,            ///< Keyboard Delete Forward key
+        ScanEnd,               ///< Keyboard End key
+        ScanPageDown,          ///< Keyboard Page Down key
+        ScanRight,             ///< Keyboard Right Arrow key
+        ScanLeft,              ///< Keyboard Left Arrow key
+        ScanDown,              ///< Keyboard Down Arrow key
+        ScanUp,                ///< Keyboard Up Arrow key
+        ScanNumLock,           ///< Keypad Num Lock and Clear key
+        ScanDivide,            ///< Keypad / key
```

Comment:
Divide (and the following codes) are on the keypad, so should use the `ScanNumpad` prefix?

Reply:
How is it done in the key code enumeration?

Reply:
The doc about `Divide` doesn't mention the keypad at all, however it is most probably implicit. There is no prefix either.

Not having a prefix generates some slight inconsistencies, for example with `ScanNumpadEqual` (for which the prefix is used to make the difference with `ScanEqual`). Note that there's no `NumpadEqual` or equivalent in the key codes. There's also one with `ScanNumpadReturn`.

Reply:
What about just the prefix `Num`? But I guess, if we prefix, let's just use the full `Numpad`.

While one might argue that they might not be part of the numpad at all times, I think the prefix makes sense: `ScanNumpadDivide`.

Reply:
`Num` is used as prefix for `0` ... `9`, so it would be ambiguous.

> While one might argue that they might not be part of the numpad at all times, I think the prefix makes sense: `ScanNumpadDivide`.

It would also make it more obvious to which key it's referring to. However, do we also update `Divide` in the key codes to `NumpadDivide`?

Reply:
Consistency is king, so if we change it here, we need to change there too. Let's just make sure we have one change for the deprecation and not let it trickle every now and then.

Reply:
I also propose to change all "keypad" to "numpad" in comments for additional consistency.

Reply:
@eliasdaler @LaurentGomila Verdict wasn't fully clear, should I rename them all to use Numpad?

Reply:
I change the Scancode enum to use ScanNumpadDivide, ScanNumpadPlus, ScanNumpadDecimal, ScanNumpadMultiply and ScanNumpadEnter. 
Given that scancodes are location based, I think it makes sense to use the prefix, so it's obvious where they are and so they don't get confused with other keys.
Whether or not the `Key` enum should be adjusted accordingly, I'll leave open as a discussion point.